doc/nrf/installation/recommended_versions.rst

```diff
@@ -18,79 +18,31 @@ Supported operating systems
 ***************************
 
 The |NCS| supports Microsoft Windows, Linux, and macOS for development.
-The following table shows the operating system versions that support the |NCS| tools:
+
+The following tables lists the support levels for the |NCS| firmware.
+(For OS support for toolchain management tools, see :ref:`the table at the bottom of the page <toolchain_management_tools>`.)
```

Comment:
```suggestion
For OS support for toolchain management tools, see :ref:`the table at the bottom of the page <toolchain_management_tools>`.
```
Similar to the corresponding sentence in the linked section, this could also more closely match the description that the linked section uses.